waku/node/v2/waku_types.nim

```diff
@@ -4,7 +4,7 @@ import libp2p/multiaddress,
        libp2p/peerinfo,
        standard_setup
 
-type WakuProto* = ref object of LPProtocol
+type WakuRelayProto* = ref object of LPProtocol
```

Comment:
Btw @decanus I noticed this file wasn't edited for your PRs, any specific reason? Seems like there'd be some protocol state, should probably be here I think?

Reply:
@oskarth to me it makes more sense currently to have the specific protocols architecturally in their own files with the protocol implementation.

Reply:
Consistency matters more. If we want to change this, then these should be moved out as well. I noticed this is how libp2p/Nimbus was structured (IIRC?), so that's why it was like this in the first place.

Maybe @kdeme knows more. Happy to go either way as long as we stay consistent.